src/test/java/study/step1/StringTest.java

```diff
@@ -0,0 +1,52 @@
+package study.step1;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+public class StringTest {
+
+    @Nested
+    class requirements1 {
+        @Test
+        void splitString1() {
+            String input = "1,2";
+            String[] result = input.split(",");
+            assertThat(result).containsExactly("1", "2");
+        }
+
+        @Test
+        void splitString2() {
+            String input = "1";
+            String[] result = input.split(",");
+            assertThat(result).containsExactly("1");
```

Comment:
```suggestion
            assertThat("1".split(",")).containsExactly("1");
```
로컬 변수 없이 구현해도 되지 않을까?